### ISSUE TITLE
Nifty is not really Nifty

### ISSUE THREAD

**Issue by Andromander** [Pannoniae/nifty#1]
Hello, I think I've found a few problems with the mod without needing to play it.

First of all, you don't really seem to be versed with the "Mod Trinity" meme. It's supposed to be "Nifty is a mod by Androsa" yeah?

Second of all, this isn't Nifty. Last I checked, I added blocks, nothing more. Also last I checked it didn't have health bars over the top of other entities and wasn't a Client-only mod.

Thirdly, and this is now a bit more serious, a number of users on Vazkii's server (the mod author of Neat, as per the meme) and myself have noted the prevalence of inconsistencies and the license is incorrect, such as "Nifty is a mod by Pannoniae", or **this glaring discrepancy in [this line](https://github.com/Pannoniae/nifty/blob/master/src/main/resources/fabric.mod.json#L6)**. Searching through your source, you basically took all of the source Vazkii had wrote on his mod, made some changes so that it doesn't look like it was stolen, and then repackaged it for Fabric. I also suspect that permission was not asked to port Neat to Fabric? So now I'm left to suspect, did you use the name of my mod - due to the license I chose - to """cleverly""" get around porting Neat to Fabric without needing to ask permission? I could be wrong, but that's what I'm getting at here. On that note, I suspect MIT is not compatible with CC A-NC-SA 3.0, let alone LGPL v3.

TL;DR: Nifty is a reused name by Androsa, is a ripoff of Neat by Vazkii, which displays health bars, stolen by Panno.

**Comment by Pannoniae** [comment]
Hi! About the names, I can't say anything; I meant it to be a pun on Neat (the mod by Vazkii).
No, the mod is not stolen, as you can see (my nearly zero features, unlike his), and you can check the source files for both mods at GitHub, I didn't even look at his code. (entirely different modloader |Fabric vs Forge|, entirely different mappings |MCP vs yarn|, would spend more time porting than rewriting from scratch)

About the "NeatMod" line, yeah, I started the mod with that name, but realized that would confuse users/others, and to be honest, would be a total misrepresentation.

I didn't know it was your name, but in this case, I am more than happy to do whatever you want.

Have a nice day! :)
